apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/TransactionImpl.java

```diff
@@ -41,30 +54,36 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-final class TransactionImpl implements Transaction {
+public class TransactionImpl implements Transaction {
     private static final Logger LOG = LoggerFactory.getLogger(TransactionImpl.class);
     private static BlockchainConfig blockchainConfig = CDI.current().select(BlockchainConfig.class).get();
 
+    @Inject
+    private BlockchainImpl blockchain;
```

Comment:
Can we make is static? Blockchain is only one for all transactions and doesnt depend on TransactionImpl Instance

Reply:
O, we can do that here. Changing...